tests/unit/PeopleAndRoles.spec.ts

```diff
@@ -88,65 +87,52 @@ describe('People And Roles component', () => {
   })
 
   it('shows all 3 add buttons when people list is empty', () => {
-    resetStore()
+    store.state.stateModel.peopleAndRoles.orgPeople = []
```

Comment:
Sorry to remove the helper method here, but I think this is more explicit and easier to understand.